tests/functional/reporting-non-auth.js

```diff
@@ -155,14 +151,14 @@ define(
             .findByCssSelector("#steps_reproduce")
             .click()
             .end()
-            .execute(function() {
-              var elm = document.querySelector("#steps_reproduce");
-              WindowHelpers.sendEvent(elm, "input");
-            })
+            // .execute(function() {
+            //   var elm = document.querySelector("#steps_reproduce");
+            //   WindowHelpers.sendEvent(elm, "input");
+            // })
             .sleep(800)
             // using xpath because 2 errors display here
-            .findByXpath(
-              "/html/body/div/main/section/div[1]/form/div/div[1]/div[3]/div/span"
+            .findByCssSelector(
+              "div.is-error:nth-child(3) > div:nth-child(1) > span:nth-child(2)"
```

Comment:
@zoepage I am stuck with this selector. I grabbed it using firebug and it passes when I run the test locally but keeps failing on travis. any suggestions? thanks